src/serde_structs/contract_abi.rs

```diff
@@ -1,111 +0,0 @@
-use crate::services::api::contract_classes::deprecated_contract_class::EntryPointType;
-use serde::Deserialize;
-use serde_json::Value;
-use std::{collections::HashMap, fs::File, path::PathBuf};
-
-#[allow(dead_code)]
-#[derive(Deserialize, Debug)]
-struct Signature {
-    inputs: Value,
-    name: String,
-    outputs: Value,
-    #[serde(default, rename = "stateMutability")]
-    state_mutability: Option<String>,
-    #[serde(rename = "type")]
-    type_name: String,
-}
-
-// We should should consider reading all the information from the abi in the future. Right now we are not considering:
-// - type: "event"
-// - type: "struct"
-pub fn read_abi(abi_name: &PathBuf) -> HashMap<String, (usize, EntryPointType)> {
```

Comment:
Im sure the `cli` crate uses this function, and it will break once updated if we remove it

Reply:
I also suggest setting the `starknet_in_rust` dependency in the `cli` crate with a relative path  to keep it up to date and prevent this kind of conflicts in the future